include/linux/soundwire/sdw.h

```diff
@@ -380,6 +381,7 @@ struct sdw_slave_prop {
 	struct sdw_dpn_prop *sink_dpn_prop;
 	u8 scp_int1_mask;
 	u32 quirks;
+	bool is_scda;
```

Comment:
typo? should be is_sdca?

Reply:
I am curious why ``@is_sdca`` at line 360 is correct. :) I always do it by copy and paste.

Reply:
oops. will fix...